apps/super-app/src/common/components/navbar/NavIcon.tsx

```diff
@@ -0,0 +1,17 @@
+import { NavLink } from "react-router-dom";
+function NavIcon(props:any) {
+  return (
+    <div>
+      <NavLink
+        to={props.link}
+        className={({ isActive }) =>
+          isActive ? "navbar-icon-active" : "navbar-icon"
```

Comment:
Instead of switching between two classes, let's switch between one class and drop the function like so:

```
className={`navbar-icon${isActive ? ' navbar-icon-active' : ''}}
```

This will simplify the CSS too (see that comment for more explanation).